dask_ml/model_selection/_hyperband.py

```diff
@@ -0,0 +1,575 @@
+from copy import deepcopy
+import logging
+import math
+
+import numpy as np
+from sklearn.base import clone
+from sklearn.model_selection import ParameterSampler
+from sklearn.utils import check_random_state
+from sklearn.metrics.scorer import check_scoring
+from tornado import gen
+import toolz
+
+import dask.array as da
+from dask.distributed import as_completed, default_client, futures_of
+from distributed.metrics import time
+from dask_searchcv.model_selection import DaskBaseSearchCV
```

Comment:
Now that #241 is in, you'll need to merge in master and change this import to `from ._search import DaskBaseSearchCV`